### DISCUSSION TITLE
MetopA satellite model availability / file location

### DISCUSSION THREAD

**Discussion by Bemtgenj** [groops-devs/groops#205] [Q&A]
Hello,

I want to process some reduced dynamic orbits for the MetopA satellite. I found the satellite relevant satellite data like kinematic orbits, satellite antennaDefintion, accuracyDefintion,... in the DATA directory  but i could not find the satellitemodel.xml file that is used in preprocessing variational equation,... script, neither could i find the file on the TUG web site. Would it be available anywhere else?

Thank you very much for the help

Cheers Jean

**Comment by bsuesser-rechberger** [comment]
Hello Jean, 
Unfortunately there is no satellite model, or at least we have not found one in the literature. However, if you find one, it would be nice if you could let us know. 
Best regards,
Barbara

**Comment by Bemtgenj** [comment]
Hello Barbara,

Txs for the quick reply

So if i understand correctly then in your reduced dynamic orbit processing you did not use a satellite model and with that for example no solar radiation forces (as no model)....

No problem - if i come across one, i will share it with you.

Cheers 
Jean

**Comment by bsuesser-rechberger** [comment]
Hello Jean, 

Yes, that's right. We introduce an acceleration bias parameter in the least squares fit to absorb the non-conservative forces.
Best regards,
Barbara

**Comment by Bemtgenj** [comment]
Hello Barbara

Txs for the feedback I was thinking of starting with the following set up. I am not sure what exactly accBias / constant with no interval does   compared to accBias with constant interval (in this case uniform sampling every 6 min) Is the first one all the time then? 

Cheers 
Jean
![accbias](https://github.com/groops-devs/groops/assets/100558965/e53ad96c-4027-46dc-a343-0ec0914a92ec)

**Comment by bsuesser-rechberger** [comment]
Hello Jean, 

if  you choose constant without any interval, than you estimate only one parameter for whole time. If you define intervals (e.g. 6 minutes), than every 6 minutes an acc. Bias is estimated and it is constant for the 6 minutes interval. But you would have to try out what the optimal settings are for your case.

Best regards,
Barbara

**Comment by Bemtgenj** [comment]
Hallo Barbara

txs for the clarification

Cheers
Jean
